### ISSUE TITLE
slow rebuild times (~ 30 seconds)

### ISSUE THREAD

**Issue by efx** [empress/broccoli-static-site-json#23]
While running the following in the [guides-source](https://github.com/ember-learn/guides-source), `BroccoliStaticSiteJson` takes an unusually long time on my machine. I have observed this with and without disabling `EMBER_CLI_SYSTEM_TEMP` which was a total shot in the dark. I can't investigate this now, but thought it would be helpful to open the issue in case anyone else experiences this.
```
$ FASTBOOT_DISABLED=true process.env.EMBER_CLI_SYSTEM_TEMP=false ember serve
Circular dependency: ../../../../../../var/folders/r6/1tb0c_rn22xdhm39d62y6rfh0000gn/T/broccoli-16461FemckdhmCPyD/cache-158-rollup/build/-private/system/model/model.js -> ../../../../../../var/folders/r6/1tb0c_rn22xdhm39d62y6rfh0000gn/T/broccoli-16461FemckdhmCPyD/cache-158-rollup/build/-private/system/model/internal-model.js -> ../../../../../../var/folders/r6/1tb0c_rn22xdhm39d62y6rfh0000gn/T/broccoli-16461FemckdhmCPyD/cache-158-rollup/build/-private/system/references.js -> ../../../../../../var/folders/r6/1tb0c_rn22xdhm39d62y6rfh0000gn/T/broccoli-16461FemckdhmCPyD/cache-158-rollup/build/-private/system/references/belongs-to.js -> ../../../../../../var/folders/r6/1tb0c_rn22xdhm39d62y6rfh0000gn/T/broccoli-16461FemckdhmCPyD/cache-158-rollup/build/-private/system/model/model.js

Build successful (57887ms) – Serving on http://localhost:4200/



Slowest Nodes (totalTime => 5% )              | Total (avg)
----------------------------------------------+---------------------
BroccoliStaticSiteJson (32)                   | 23814ms (744 ms)
Package /assets/vendor.js (1)                 | 4004ms
Rollup (1)                                    | 3195ms
BroccoliMergeTrees (22)                       | 3162ms (143 ms)
Babel: ember-guides (4)                       | 2971ms (742 ms)

file changed models/customizing-serializers.md

Build successful (29783ms) – Serving on http://localhost:4200/



Slowest Nodes (totalTime => 5% )              | Total (avg)
----------------------------------------------+---------------------
BroccoliStaticSiteJson (32)                   | 25009ms (781 ms)
ember-auto-import-analyzer (4)                | 2702ms (675 ms)

file deleted 4913
file changed index.md

Build successful (29558ms) – Serving on http://localhost:4200/



Slowest Nodes (totalTime => 5% )              | Total (avg)
----------------------------------------------+---------------------
BroccoliStaticSiteJson (32)                   | 24852ms (776 ms)
ember-auto-import-analyzer (4)                | 2722ms (680 ms)

```

**Comment by mansona** [comment]
Hey @efx 👋 this will have been fixed as part of the recently merged PR #32 🎉 

The update is making it's way downstream to the Ember Guides as we speak https://github.com/empress/guidemaker/pull/32 👍

**Comment by efx** [comment]
woot, thanks @mansona !
